shoes-swt/lib/shoes/swt/generate_backend.rb

```diff
@@ -1,7 +1,35 @@
 require 'rbconfig'
```

Comment:
Will add this one, but I'm thinking it'll be worth doing #1284 soon and then enabling Rubocop to enforce going forward.